hw/top_englishbreakfast/util/sw_sources.patch

```diff
@@ -27,7 +27,7 @@ index 8861f54ba..8442bb896 100644
 --- a/sw/device/lib/pinmux.c
 +++ b/sw/device/lib/pinmux.c
 @@ -60,24 +60,4 @@ void pinmux_init(void) {
-   reg_offset = kTopEarlgreyPinmuxMioOutIoc11 << 2;
+   reg_offset = kTopEarlgreyPinmuxMioOutIoc4 << 2;
```

Comment:
I'm not sure, but isn't this patch missing the `Ioc10 -> Ioc3` change?

Reply:
Thanks @msfschaffner for the review!
That's a good question, I'm assuming that the RX is not important for the `english_breakfast` target test.
But I want to hear @vogelpi option.

Reply:
I can confirm that `english_breakfast` needs both RX and TX to communicate with the capture/scope board.

Reply:
I looked at the patch again and I understood what it really does for the `sw/device/lib/pinmux.c`. It just removes the UART1 from the `Earlgray` target and It is not changing anything related to UART0 as you can see below:
![image](https://user-images.githubusercontent.com/92785679/144045797-d7e5cd9c-be91-42eb-b5f7-64295a20eb05.png)

So, this PR doesn't need to change anything else on the `sw_sources.patch` file.